repository-data/application/src/main/resources/hcm-actions.yaml

```
@@ -117,3 +117,7 @@ action-lists:
     /content/documents/administration/bsh-labels/modules/events-listings-module: reload
 - 2.9.1:
     /content/documents/administration/configuration/default-site-properties: reload
+- 2.9.2:
+      /content/documents/administration/labels/modules/map: reload
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (2 > 0)

(empty-lines)